src/plugin.spec.ts

```diff
@@ -10,22 +10,11 @@ import {
   type UpdateSnapshotOptions
 } from './plugin'
 
-function deleteFileSafely(filePath: string): void {
-  try {
-    unlinkSync(filePath)
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } catch (err: any) {
-    if (err.code !== 'ENOENT') {
-      throw err // If the error is not "File not found" (ENOENT), rethrow it
-    } // If the error is "File not found," do nothing (fail silently)
-  }
-}
-
 const baseUpdateOptions: UpdateSnapshotOptions = {
   screenshotName: 'enjuto',
   specName: 'sub-folder',
-  screenshotAbsolutePath: path.join('fixtures', 'assets', 'base', 'enjuto.png'),
-  baseDirectory: path.join('fixtures', 'assets', 'base'),
+  screenshotAbsolutePath: path.join('tests/integration/cypress/fixtures/assets/base/enjuto.png'),
```

Comment:
Is this a good idea?
I have my doubts about that this will work in all OS.

Reply:
This is what I wanted to put to test with the latest platform checks, but then I started getting a really weird problems with .png images getting unrecognized, could not find my way around it still

Reply:
Ok, seems to be working just fine on all platforms, so I'd say it's ready for another review @area73

Reply:
Done !